config/webpacker.yml

```diff
@@ -16,7 +16,7 @@ default: &default
   cache_manifest: false
 
   # Extract and emit a css file
-  extract_css: true
+  extract_css: false
```

Comment:
Was this change intentional?

Reply:
Yeah, it causes issues for rspec (think you need to compile assets before running tests?) so wanted to revert this until checking it properly. Left the `inline: false` for dev setting though.

Reply:
👍